src/useGrid.js

```diff
@@ -196,9 +196,11 @@ export default function useGrid({ locale, month: currentMonth, onMonthChange, tr
         }
       }
 
-      containerElement.addEventListener('touchstart', handleDragStart)
-      containerElement.addEventListener('touchmove', handleDrag)
-      containerElement.addEventListener('touchend', handleDragEnd)
+      if (touchDragEnabled) {
```

Comment:
Might be good to move this check to the beginning of the `useLayoutEffect`'s function (line 123) so the rest of it doesn't get executed, like:

```
if (!touchDragEnabled) {
  return
}
```